README.md

```diff
@@ -43,6 +43,43 @@ async def main():
         async with client.get('https://ya.ru') as response:
             print(response.status)
 ```
+
+```python
```

Comment:
I think that example of trace_config_ctx is unnecessary in README. It's advanced logic. You can put this example in as test in tests or create directory examples. So, in readme put something like this: "you can see request attempt in trace, check example here".